### ISSUE TITLE
bug: Logto CLI "SyntaxError: The requested module '@logto/shared' does not provide an export named 'GlobalValues'"

### ISSUE THREAD

**Issue by hyenabyte** [logto-io/logto#3568]
### Describe the bug
I am getting the following error when trying to seed my db:

```
file:///home/hyena/.local/share/pnpm/global/5/.pnpm/@logto+cli@1.0.0-rc.3/node_modules/@logto/cli/lib/commands/database/seed/cloud.js:2
import { GlobalValues } from '@logto/shared';
         ^^^^^^^^^^^^
SyntaxError: The requested module '@logto/shared' does not provide an export named 'GlobalValues'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)

Node.js v18.14.1
```

Any use of the Logto CLI results in this error, both when installing globally or running `npx init @logto/cli` or `pnpx create @logto/cli`.

I have tested with both my M1 Mac and a VPS running NixOS.

### Expected behavior
Normal use of the Logto CLI

### How to reproduce?
Run any `logto` command, e.g. `logto --version`

### Context

OS: MacOS / NixOS

Environment: Local

Logto version: 1.0.0-rc.3

Node version: 18.x and 19.x

Postgres version: n/a

Command of starting Logto: `logto db seed`

**Comment by gao-sun** [comment]
sorry, there was some misconfiguration so the CLI didn't publish to NPM

**Comment by gao-sun** [comment]
just published a new version. would you like to try again?

**Comment by hyenabyte** [comment]
It seems to have resolved the error, however I can't seem to install, I get this error:
```
 ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @logto/schemas@1.0.0

This error happened while installing the dependencies of @logto/cli@1.0.1

The latest release of @logto/schemas is "1.0.0-rc.1".
```

**Comment by gao-sun** [comment]
so weird. i don't know why npm says this, i'll force bump a version for that package

<img width="1105" alt="image" src="https://user-images.githubusercontent.com/14722250/226592700-6aa10f36-3a2a-40ce-aea8-c300b4bc0d05.png">

**Comment by hyenabyte** [comment]
I still get the same problem, I dont know if the version bump got published
```
$ pnpm view @logto/schemas versions
[
  '1.0.0-beta.11',
  '1.0.0-beta.12',
  '1.0.0-beta.13',
  '1.0.0-beta.14',
  '1.0.0-beta.15',
  '1.0.0-beta.16',
  '1.0.0-beta.17',
  '1.0.0-beta.18',
  '1.0.0-rc.0',
  '1.0.0-rc.1'
]
```

**Comment by deepziyu** [comment]
> It seems to have resolved the error, however I can't seem to install, I get this error:
> 
> ```
>  ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @logto/schemas@1.0.0
> 
> This error happened while installing the dependencies of @logto/cli@1.0.1
> 
> The latest release of @logto/schemas is "1.0.0-rc.1".
> ```
 Got the same npm ERR report.

**Comment by gao-sun** [comment]
npm really confuses me. as you can see:

https://www.npmjs.com/package/@logto/schemas?activeTab=versions

there's no 1.0.0 and 1.0.1, but the [CI](https://github.com/logto-io/logto/actions/runs/4487217596/jobs/7890425482) says: 

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/14722250/226839763-132e1eae-726e-44b6-82fb-b599db05e011.png">

(this also happened to v1.0.0) i'm not sure if we met the same situation as https://github.com/npm/cli/issues/5058

i even tried to publish schemas locally, still the same error

**Comment by gao-sun** [comment]
it should be fixed by #3579. i can use `pnpm create` now:

<img width="665" alt="image" src="https://user-images.githubusercontent.com/14722250/226844307-3d2e8904-19ad-4ef9-8ff2-0dcd26690c75.png">

**Comment by hyenabyte** [comment]
It works! Thank you for the help 🙌
